pages/common/aws-secretsmanager.md

```diff
@@ -0,0 +1,32 @@
+# aws secretsmanager
+
+> Store, manage, and retrieve secrets.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/secretsmanager/>.
+
+- Show secrets stored by the secrets manager in the current account:
+
+`aws secretsmanager list-secrets`
+
+- Create a secret:
+
+`aws secretsmanager create-secret --name {{name}} --description "{{String describing secret}}" --secret-string {{secret}}`
```

Comment:
The placeholders are variables in `snake_case`
```suggestion
`aws secretsmanager create-secret --name {{name}} --description "{{secret_description}}" --secret-string {{secret}}`
```

Reply:
The description can have spaces though.